libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -185,64 +178,64 @@ irods::error start(irods::default_re_ctx& _u,const std::string& _instance_name)
 }
 
 irods::error stop(irods::default_re_ctx& _u,const std::string& _instance_name) {
+    (void) _u;
 
     std::lock_guard<std::mutex> lock(audit_plugin_mutex);
 
-    json_t* obj = json_object();
-    if( !obj ) {
-        return ERROR(SYS_MALLOC_ERR, "json_object() failed");
+    irods::error ret = get_re_configs( _instance_name );
+    if( !ret.ok() ) {
+        irods::log(PASS(ret));
     }
 
+    messenger = pn_messenger(NULL);
+    pn_messenger_start(messenger);
+    pn_messenger_set_blocking(messenger, false);  // do not block
```

Comment:
Why are these lines being added?

Reply:
Thoughtless copy-pasting -- you're right that the messenger is static so reinitializing is silly.